### PR TITLE
Add diagram to README.

### PR DESCRIPTION
**Pull request by PEConn**
Adds a diagram that shows how all the commands and files relate to each other.

It also contains the GraphViz files I used to generate that diagram. We don't have a way of automatically generating it yet, but it should be updated infrequently enough that manually updating it should work.

### CODE REVIEW

File: packages/cli/command_flow.dot [hunk 1]
```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
```
Comment (andreban) [inline packages/cli/command_flow.dot:2]: nit: 2020

```suggestion
 * Copyright 2020 Google Inc. All Rights Reserved.
```

File: packages/cli/README.md [hunk 2]
```diff
@@ -110,12 +110,16 @@ explains how to extract the information needed to generate it.
 
 ## Commands
 
+![Overview of bubblewrap commands](command_flow.svg)
+
+The above diagram shows a how all of the commands (in the black boxes) and the artifacts (in the while ovals) effect each other.
```
Comment (andreban) [inline packages/cli/README.md:115]: typo

```suggestion
The above diagram shows how all of the commands (in the black boxes) and the artifacts (in the while ovals) effect each other.
```

**Review by andreban — CHANGES_REQUESTED** [review]

**Comment by PEConn** [comment]
Done

**Review by andreban — APPROVED** [review]
LGTM, Thank you!

**Pull request merged**